faithcomesbyhearing/dbp
# FCBHDBP-621 Fix the endpoint api/bibles/:bible_id/book to display the verse_count for Old Testament (OT) books.

# Description
Fix logic to select the plain text fileset related to verses when the Bible has one fileset for the New Testament (NT) and another for the OT

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-621

## How Do I QA This
You should run the following postman test and it has to pass:

https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-423cb4ed-be7a-431e-9b35-80512e00fc1e?action=share&source=copy-link&creator=17122915&active-environment=810fd94b-9392-43e2-9f13-943e8d323135
